### DISCUSSION TITLE
Predefined Colors

### DISCUSSION THREAD

**Discussion by damianday** [godotengine/godot-proposals#8396] [Scripting]
It would be really fabulous if we had some predefined colors available for the Color class, it's a simple addition that everybody would benefit from. In my C# scripts I'm not able to extend the color class with properties so can't make an extension, but anyway I feel it would better be suited to being in the core class.

**Comment by Jesusemora** [comment]
Read the documentation, There's like 200 predefined colors:

https://docs.godotengine.org/es/stable/classes/class_color.html

**Reply by AThousandShips** [reply]
The English documentation:
https://docs.godotengine.org/en/stable/classes/class_color.html

**Reply by AThousandShips** [reply]
See [here](https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_differences.html#color) for accessing them in C#

**Reply by damianday** [reply]
Ah brilliant thanks guys. Think it would be more intuitive to have them be part of the colour class for C# but meh
